service/service.go

```diff
@@ -241,7 +263,7 @@ func (s *service) logCsiNodeTopologyKeys() error {
 		K8sClientset = k8sutils.Clientset
 	}
 
-	csiNodes, err := K8sClientset.StorageV1().CSINodes().List(context.TODO(), metav1.ListOptions{})
+	csiNodes, _ := K8sClientset.StorageV1().CSINodes().List(context.TODO(), metav1.ListOptions{})
```

Comment:
**suggestion:** Instead of removing the `err`, why not add a check?

Reply:
Will address this.

Reply:
fixed